frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -44,6 +44,25 @@
       "display_name": "workerman postgres",
       "notes": "",
       "versus": "php"
+    },
+    "async": {
+      "db_url": "/db",
+      "fortune_url": "/fortune",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "MySQL",
+      "framework": "None",
+      "language": "PHP",
+      "flavor": "PHP7",
+      "orm": "Raw",
+      "platform": "workerman",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "workerman async db",
```

Comment:
Every display name so far seems to conform to hyphens rather than spaces - `workerman-async-db`

Reply:
ready